packages/storefront/src/views/Page.vue

```diff
@@ -101,6 +129,19 @@
         await this.$router.replace({ name: '404' });
       }
     }
+
+    public get isLoading(): boolean {
+      return this.$store.getters.isLoading;
+    }
+
+    public get storefrontTheme(): StorefrontTheme {
+      return this.$store.getters.storefrontTheme;
+    }
+
+    // getter is not called again on this.$forceUpdate()
```

Comment:
`getSkeletonTheme()` is not a getter

Reply:
Improved comment to 
```
// needs to real function, because a getter like `public get skeletonTheme()` is not called again on this.$forceUpdate()
```